.github/environments/enabled/config.yml

```diff
@@ -0,0 +1,32 @@
+DRYDOCK_INIT_JOBS: true
+K8S_NAMESPACE: testing-openedx
+PLUGINS:
+- forum
+- mfe
+- drydock
+- s3
```

Comment:
You had a test for s3_disable and you deleted it. In this case, since you're using S3 in the "enabled" case, you could a different test called minio_enabled to check the case when using MinIO and not S3